src/libcore/convert.rs

```diff
@@ -17,6 +17,26 @@
 //! Like many traits, these are often used as bounds for generic functions, to
 //! support arguments of multiple types.
 //!
+//! - Use `as` for reference-to-reference conversions
```

Comment:
So yes, I think that all we need is proper capitalization/style here, to disambiguate from the keywords.